### ISSUE TITLE
Keyboard navigation doesn't work in Docs

### ISSUE THREAD

**Issue by clibu** [PostHog/posthog#46207]
### Bug Description

PgUp, PgDn, Home, End etc. don't work in the Docs.

## How to reproduce

1. Open any Doc page: ex. https://posthog.com/docs/libraries/js/usage
2. Press PgUp, PgDn, Home, End and nothing happens

Further Shift + Cursor keys don't select text and instead move the window.

## Additional context

Tried in Chrome Version 144.0.7559.97 (Official Build) (64-bit) and Edge Version 144.0.3719.92 (Official build) (64-bit) on Windows 11

### Debug info

```shell
- [ ] PostHog Cloud, Debug information: [please copy/paste from https://us.posthog.com/settings/project-details#variables or https://eu.posthog.com/settings/project-details#variables]
- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see [`Sunsetting Kubernetes support`](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]
```

**Comment by vmaineng** [comment]
Hello! 

I looked into this keyboard navigation issue and I noticed that:

1. The bug occurs on **posthog.com/docs** (the docs/marketing site)
2. This issue is filed in the **posthog/posthog** repo (the product app)


Since this bug is on posthog.com/docs, should this issue be in the posthog.com repo instead?

Please let me know what your thoughts are.

**Comment by clibu** [comment]
Hi Mai, I'm new to PostHog and didn't realise there was a different Github site for docs etc. I assume I should have posted this on: https://github.com/PostHog/posthog.com/issues 

What do you suggest is the best way forward? Can it be moved, deleted and reposted or what?
